test/python/test_quantumprogram.py

```diff
@@ -690,12 +691,13 @@ def test_compile_program(self):
         out = QP_program.compile(['circuitName'], backend=backend,
                                  coupling_map=coupling_map, qobj_id='cooljob')
         self.log.info(out)
-        self.assertEqual(len(out), 3)
+        self.assertEqual(out.id_, 'cooljob')
```

Comment:
XDDDDDD cool! :D

Reply:
Actually the credit for the variable should go to Jay https://github.com/QISKit/qiskit-sdk-py/commit/858487b8f1c6083a54ab50a61d14a03693673e88 :cool: :D